SYCL/ESIMD/api/functional/logger.hpp

```diff
@@ -20,22 +20,19 @@
 namespace esimd_test {
 namespace api {
 namespace functional {
```

Comment:
```suggestion
namespace esimd_test::api::functional {
```
While we are here :)

Reply:
I've applied this comment in this commit: https://github.com/intel/llvm-test-suite/pull/614/commits/d22d5e27f2e4c39195c318acbf85851734ad9ef9

Reply:
Great, thanks for the applying to all files as well :)